### PR TITLE
Changed text color for all error messages boxes for coin_balances page to the required one / Main Blockscout page banner height bug fix and new changes / New tooltips for Block Details page added / Token image added to the Token instance page.

### PR DESCRIPTION
**Pull request by YegorSan**
* Issues were introduced at
 * #2621
 * #1771 
 * **This issue wasn't introduced yet** :   
  Fixed height issue with top-banner on the main Blockscout page.  

## Motivation

* #2621 - Error box text colors has different style.
* #1771 - New Tooltips were needed for the Block Details page. 

* Top banner on the Blockscout home page is opening with the height bug now. I've added several changes in order to fix this issue (fixed height etc.) .

### Enhancements
*  Changed style for all error messages blocks to required one. Added "cursor: pointer" and switched "a href=" to "span" (let me know if it's not required) , as I guess all of the error messages of such a type should have the same style. (one error block has "a href=" another one - span).

* I've commented "display: none" for error messages block in devtools  just in order to show you performed changes and you can see both messages now on the following gif:

Sokol-Testnet-Explorer-BlockScout
* #1771 - Added tooltips for the Block Details Page (Additional logic is required for Miner/Validator tooltip text and Block Rewards container. ) : 
Block-10428651-Sokol-Testnet-Explorer-_6_

### Bug Fixes
Fixed height bug with top banner container on the main BlockScout Page:

* **Was:**

 Запись_экрана_2019-08-14_в_19 50 21

* **Now:** 

AwesomeScreenshot-2019-8-20-1566309626893
## Checklist for your PR

  - [ ] I added an entry to `CHANGELOG.md` with this PR
  - [ ] If I added new functionality, I added tests covering it.
  - [ ] If I fixed a bug, I added a regression test to prevent the bug from silently reappearing again.
  - [ ] I checked whether I should update the docs and did so if necessary
  - [ ] If I added/changed/removed ENV var, I should update the list of env vars in https://github.com/poanetwork/blockscout/blob/master/docs/env-variables.md to reflect changes in the table here https://poanetwork.github.io/blockscout/#/env-variables?id=blockscout-env-variables. I've set `master` in the `Version` column.
  - [ ] If I add new indices into DB, I checked, that they don't redundant with PGHero or other tools

### CODE REVIEW

**Comment by ayrat555** [comment]
why did you close the previous PR with the same changes?

**Comment by vbaranov** [comment]
@ayrat555 we had a call with Egor. I've asked to repair his PRs because they were created from the same branch and were branched from a wrong branch in the main repo.

**Comment by YegorSan** [comment]
@vbaranov 

I've re-checked all the files and everything seems to be fine. I had some merge conflicts in my local branch and  it took some time to resolve issues, merge changes between branches, and to re-check everything. I've tried to run command for internationalization and it was executed successfully without errors, but still as far as I can see _gettext_ tests were failed. Please clarify how to correctly add a new entry to Changelog.md. Thank's in advance.

**Comment by YegorSan** [comment]
@vbaranov 
I am sure issue is related to the merging conflicts in my local repo. I'll try to resolve it in the shortest terms.

**Comment by vbaranov** [comment]
@YegorSan make sure to actualize your branch with updates from *front-update* (I merged upstream from *master* branch to *front-update*)

**Comment by YegorSan** [comment]
@vbaranov
Looks like i've resolved merge all conflicts. Could you please make a review of changes , and let me know if something is still wrong. I am locally on a frontend-tasks branch now, and looks like everything is working fine. I've also fixed gettext issue, and really hope other issues are finally resolved too.

**Comment by YegorSan** [comment]
@vbaranov
I want to add that in the morning I've worked on #2641 issue and fixed colors for the mentioned buttons. And these changes are coming with the current updates.

File: apps/block_scout_web/assets/css/components/_erc721_token_image_container.scss [hunk 1]
```diff
@@ -0,0 +1,10 @@
+/* ERC721 image block */
+.erc721-image {
```
Comment (vbaranov) [inline apps/block_scout_web/assets/css/components/_erc721_token_image_container.scss]: @YegorSan please remove it from this PR. It doesn't relate to issues you are solving

File: apps/block_scout_web/assets/css/app.scss [hunk 2]
```diff
@@ -13,13 +15,10 @@ $fa-font-path: "~@fortawesome/fontawesome-free/webfonts";
 @import "fa-brands";
 @import "fa-regular";
 @import "fa-solid";
-
```
Comment (vbaranov) [inline apps/block_scout_web/assets/css/app.scss]: @YegorSan are these changes in this stylesheet necessary?

File: apps/block_scout_web/assets/css/components/_transaction.scss [hunk 3]
```diff
@@ -1,46 +1,53 @@
 .transaction-bottom-panel {
-	display: flex;
-	flex-direction: column;
-	@media (min-width: 768px) {
-		flex-direction: row;
```
Comment (vbaranov) [inline apps/block_scout_web/assets/css/components/_transaction.scss]: @YegorSan it looks like these changes are unnecessary

File: apps/block_scout_web/lib/block_scout_web/templates/block/overview.html.eex [hunk 4]
```diff
@@ -4,18 +4,24 @@
...
       <div class="card card-mr-50-md">
         <div class="card-body">
-          <h1 class="card-title" data-test="detail_type">
+          <h1 class="card-title" data-test="detail_type" >
             <%= gettext("%{block_type} Details", block_type: block_type(@block)) %>
           </h1>
           <!-- Block Height -->
-          <h3 data-test="block_detail_number">
+          
+
```
Comment (vbaranov) [inline apps/block_scout_web/lib/block_scout_web/templates/block/overview.html.eex]: @YegorSan do we need empty lines here?

File: apps/block_scout_web/lib/block_scout_web/templates/block/overview.html.eex [hunk 5]
```diff
@@ -91,20 +106,19 @@
                 </dd>
               </dl>
             <% end %>
-
             <!-- Otherwise it will be displayed in its own block -->
             <%= if show_reward?(@block.rewards) do %>
-              <dl class="row">
+              <dl class="row" >
```
Comment (vbaranov) [inline apps/block_scout_web/lib/block_scout_web/templates/block/overview.html.eex]: @YegorSan let's remove this unnecessary whitespace

File: apps/block_scout_web/lib/block_scout_web/templates/block/overview.html.eex [hunk 6]
```diff
@@ -133,13 +150,17 @@
...
         <div class="card-body card-body-flex-column-space-between">
+        
           <%= if show_reward?(@block.rewards) do %>
             <h2 class="card-title balance-card-title"><%= gettext "Block Rewards" %></h2>
-            <div class="text-right">
+            <div class="text-right" style="margin-left: 50%;" data-toggle="tooltip" data-placement="top" title="" data-original-title="Amount of distributed reward. Miners receive a static block reward + Tx fees + uncle fees.">
+                            
               <%= for block_reward <- @block.rewards do %>
-                <p class="address-current-balance"><%= block_reward_text(block_reward) %> <span class="text-muted"><%= format_wei_value(block_reward.reward, :ether) %></span></p>
+
```
Comment (vbaranov) [inline apps/block_scout_web/lib/block_scout_web/templates/block/overview.html.eex]: @YegorSan again, unnecessary line break

File: apps/block_scout_web/lib/block_scout_web/templates/block/overview.html.eex [hunk 7]
```diff
@@ -150,6 +171,7 @@
                 <span class="text-muted">(<%= (Decimal.to_integer(@block.gas_used) / Decimal.to_integer(@block.gas_limit)) |> BlockScoutWeb.Cldr.Number.to_string!(format: "#.#%") %>)</span>
               </h3>
               <p class="address-current-balance"><%= @block.gas_limit |> BlockScoutWeb.Cldr.Number.to_string! %><%= gettext "Gas Limit" %></p>
+
```
Comment (vbaranov) [inline apps/block_scout_web/lib/block_scout_web/templates/block/overview.html.eex]: the same as above

File: apps/block_scout_web/lib/block_scout_web/templates/tokens/overview/_details.html.eex [hunk 8]
```diff
@@ -71,7 +71,7 @@
     <%= if total_supply?(@token) do %>
       <div class="card-section col-md-12 col-lg-4 pl-0-md">
         <div class="card card-background-1">
-          <div class="card-body">
+          <div class="card-body">
```
Comment (vbaranov) [inline apps/block_scout_web/lib/block_scout_web/templates/tokens/overview/_details.html.eex]: please remove

File: apps/block_scout_web/lib/block_scout_web/templates/transaction_internal_transaction/index.html.eex [hunk 9]
```diff
@@ -23,4 +23,10 @@
       <%= render BlockScoutWeb.CommonComponentsView, "_pagination_container.html", position: "bottom", cur_page_number: "1", show_pagination_limit: true, data_next_page_button: true, data_prev_page_button: true %>
     </div>
   </div>
+
+
```
Comment (vbaranov) [inline apps/block_scout_web/lib/block_scout_web/templates/transaction_internal_transaction/index.html.eex]: ?

File: apps/block_scout_web/assets/css/app.scss (hunk 2 above)
Comment (YegorSan) [reply apps/block_scout_web/assets/css/app.scss]: @vbaranov 
When I've started to work, I've added styles straightly inside app.scss file when worked with updates.

Comment (YegorSan) [reply apps/block_scout_web/assets/css/app.scss]: @vbaranov 
But I am sure I didn't touch anything. I've only added stylesheets below the already written code.

File: apps/block_scout_web/assets/css/components/_transaction.scss (hunk 3 above)
Comment (YegorSan) [reply apps/block_scout_web/assets/css/components/_transaction.scss]: @vbaranov 
I don't actually understand how it's was happened. I am always using text formatter once I finishing with the code updates. As I see there was simply deleted and then added again the same content.

File: apps/block_scout_web/assets/css/app.scss (hunk 2 above)
Comment (vbaranov) [reply apps/block_scout_web/assets/css/app.scss]: @YegorSan please remove all mentioned unnecessary changes. It is very hard to review PR having those changes. Check this diff page https://github.com/poanetwork/blockscout/pull/2637/files to find what you are trying to unintentionally change.

File: apps/block_scout_web/assets/css/components/_transaction.scss (hunk 3 above)
Comment (vbaranov) [reply apps/block_scout_web/assets/css/components/_transaction.scss]: @YegorSan not the same. You add more tabs/whitespaces. This is why these changes display here.

**Comment by YegorSan** [comment]
@vbaranov 

I've removed unnecessary lines, and one file which was added for ERC721 page. I even didn't know that it's so important to check with the empty spaces. By the way, I have code formatting enabled in Visual Studio Code, and I found that sometimes it can add empty spaces between blocks. I've turned it off.
